sky/provision/gcp/instance.py

```diff
@@ -228,9 +228,11 @@ def open_ports(
 
 def cleanup_ports(
     cluster_name_on_cloud: str,
+    ports: List[str],
     provider_config: Optional[Dict[str, Any]] = None,
 ) -> None:
     """See sky/provision/__init__.py"""
+    del ports
```

Comment:
ditto

Reply:
Done in https://github.com/skypilot-org/skypilot/pull/2713/commits/d3fc952568a4ddf853355b15006b42fecd737e32